src/message/MessageListItem.js

```diff
@@ -19,6 +19,8 @@ export default class MessageListItem extends PureComponent {
 
     if (type === 'time') {
       return <TimeRow key={`time${timestamp}`} timestamp={timestamp} />;
+    } else if (message.isOutbox) {
+      return <MessageContainer isBrief={isBrief} message={message} />;
```

Comment:
The function in the MessageListContainer is called very very frequently onScroll, so it can be more expensive (even though there will be very less number of outbox & visibleIds)

Is their some performance bottleneck for rendering of a different component?